recipes/pdal-c/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -ex
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+if [ "$(uname)" == "Linux" ]; then
+   export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+fi
+
+cmake -G "Unix Makefiles" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_LIBRARY_PATH=$PREFIX/lib \
+  -DCMAKE_INCLUDE_PATH=$PREFIX/include \
+  -DCONDA_BUILD=ON \
+  .
+  
+
+# CircleCI offers two cores.
+make -j $CPU_COUNT
+make install
```

Comment:
Do we have a make check we can run here?

Reply:
The short answer is no ... We don't currently have any and this recipe was adapted from the parent PDAL recipe.

What are you expecting to see?

Thanks for picking up the review.

Paul

Reply:
Just more tests if they are available. It is fine if we don't have them but if we do, let's run them.